src/main/java/io/jenkins/plugins/propelo/job_reporter/plugins/PropeloPluginImpl.java

```diff
@@ -51,7 +52,7 @@ public class PropeloPluginImpl extends Plugin {
     private static final String DATA_DIR_NAME = "run-complete-data";
     public static final String PLUGIN_SHORT_NAME = "propelo-job-reporter";
     private Secret levelOpsApiKey = Secret.fromString("");
-    private String levelOpsPluginPath = "${JENKINS_HOME}/levelops-jenkin";
+    private String levelOpsPluginPath = "${JENKINS_HOME}/levelops-jenkins";
```

Comment:
If we change this, it would affect ll existing plugins that upgrade. The path can stay as-is. Customers do not see it or navigate to it.

Reply:
done